py3-google-api-python-client.yaml

```diff
@@ -43,3 +43,27 @@ update:
   github:
     identifier: googleapis/google-api-python-client
     strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - python-3
+        - py3-google-auth
+        - py3-google-auth-httplib2
+        - py3-google-api-core
+        - py3-uritemplate
+        - py3-httplib2
```

Comment:
This test should install the package and deps so you don't need to add runtime deps here.